src/utils/Sort.js

```diff
@@ -645,158 +679,179 @@ export function cardGetLabels(card, sort) {
         ].includes(type)
       ) {
         return ['Other'];
-      } else {
-        return [type];
       }
-    } else if (colorIdentity(card).length === 5) {
+      return [type];
+    }
+    if (colorIdentity(card).length === 5) {
       return ['Five Color'];
-    } else {
-      return [
-        ...cardGetLabels(card, 'Guilds'),
-        ...cardGetLabels(card, 'Shards / Wedges'),
-        ...cardGetLabels(card, '4+ Color'),
-      ];
     }
-  } else if (sort == 'Artist') {
+    return [
+      ...cardGetLabels(card, 'Guilds'),
+      ...cardGetLabels(card, 'Shards / Wedges'),
+      ...cardGetLabels(card, '4+ Color'),
+    ];
+  }
+  if (sort === 'Artist') {
     return [card.details.artist];
-  } else if (sort == 'Legality') {
+  }
+  if (sort === 'Legality') {
     return Object.entries(card.details.legalities)
       .filter(([k, v]) => ['legal', 'banned'].includes(v))
       .map(([k, v]) => k);
```

Comment:
Here one of the functions doesn't use the key `k` and the other doesn't use the value `v`.

Reply:
Alright. I'll plan to add a comment to tell eslint to ignore those errors then.

Reply:
You can replace the variable name with `_` and it will suppress this error iirc

Reply:
That hasn't been working for me, still getting the same error with `_` and no comment.